storybook publishが壊れてる

https://github.com/misskey-dev/misskey/pull/11245#issuecomment-1631694780

```
Build and open your Storybook locally and check the browser console for errors.
Visit your published Storybook at https://6428f7d7b962f0b79f97d6e4-knlemknljj.chromatic.com/
The following error was encountered while running your Storybook:

"undefined" is not valid JSON
    → Failed to publish build
undefined
/home/runner/work/misskey/misskey/packages/frontend:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  frontend@ chromatic: `chromatic "--exit-once-uploaded" "-d" "storybook-static" "--branch-name" "pag-back"`
```

--- a/locales/index.d.ts
+++ b/locales/index.d.ts
@@ -2158,4 +2158,4 @@ export interface Locale {
 declare const locales: {
     [lang: string]: Locale;
 };
-export = locales;
+export default locales;

--- a/packages/frontend/.storybook/preload-locale.ts
+++ b/packages/frontend/.storybook/preload-locale.ts
@@ -1,5 +1,5 @@
 import { writeFile } from 'node:fs/promises';
-import * as locales from '../../../locales/index.js';
+import locales from '../../../locales/index.js';
 
 await writeFile(
 	new URL('locale.ts', import.meta.url),

